includes/Modules/PageSpeed_Insights.php

```diff
@@ -193,7 +191,7 @@ protected function setup_info() {
 	 * @return Google_Client Google client instance.
 	 */
 	protected function setup_client() {
-		return $this->authentication->get_api_key_client()->get_client();
+		return $this->authentication->get_oauth_client()->get_client();
```

Comment:
This method can also be removed since it now matches the base class implementation.